.github/workflows/dry-run.yml

```diff
@@ -0,0 +1,87 @@
+name: sync-team dry-run
+
+on:
+  workflow_run:
+    workflows: [ CI ]
+    types:
+      - completed
+
+jobs:
+  dry-run:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' && github.event.workflow_run.event == 'pull_request' }}
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: rust-lang/sync-team
+
+      - name: Install Rust Stable
+        run: |
+          rustc -vV
+          rustup update stable
+          rustup default stable
+          rustc -vV
+
+      - uses: Swatinem/rust-cache@v2
+
+      - name: Download built JSON API
+        uses: actions/download-artifact@v4
+        with:
+          name: team-api-output
+          path: team-api
+          run-id: ${{ github.event.workflow_run.id }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Run sync-team dry-run check
+        run: |
+          cargo build
+          ./target/debug/sync-team print-plan --team-source team-api > output.log
+
+      - uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
```

Comment:
This script is a bit complex. What do you think about storing the script in a separate file and invoking it from here?
Instructions [here](https://github.com/actions/github-script?tab=readme-ov-file#run-a-separate-file-with-an-async-function)

Having the script in a separate file will have various benefits, such as syntax highlight and IDE autocomplete

Reply:
I can auto-inject syntax highlight and autocomplete into the snipper in my IDE (CLion) :grin:

We would have to checkout the team repo for that. I don't have a problem with that, but it could be one additional potential attack vector.

Reply:
oooh, I see, nevermind 👍